src/mono/mono/mini/simd-intrinsics.c

```diff
@@ -1379,6 +1392,23 @@ emit_sri_vector (MonoCompile *cfg, MonoMethod *cmethod, MonoMethodSignature *fsi
 	}
 #endif
 
+#ifdef TARGET_WASM
+	g_assert (COMPILE_LLVM (cfg));
+#endif
+
+#ifdef TARGET_AMD64
+	if (!COMPILE_LLVM (cfg)) {
+		if (vector_size != 128)
+			return NULL;
+#ifdef TARGET_WIN32
+		return NULL;
```

Comment:
@vargaz if I understand this correctly, there is currently no support for `Vector128<T>` on windows-x64, is that correct?

I am asking this because I noticed `System.Runtime.Intrinsics.Tests.Vectors.Vector128Tests.*` started failing on `runtime-extra-platforms` pipeline in the following legs:
- net8.0-windows-Release-x64-Mono_Release-Windows.11.Amd64.Client.Open
- net8.0-windows-Release-x64-Mono_Release-Windows.Amd64.Server2022.Open
- net8.0-windows-Release-x64-Mono_Release-Windows.81.Amd64.Open

which was reported here: https://github.com/dotnet/runtime/issues/88983
The failures started happening on Jun-14 being the same date when this PR got merged in.

Should we disable the tests? And is there a tracking issue about the missing support?

Reply:
I agree. I was puzzled by this as well.

Reply:
Thanks, I will continue investigating.